File: bitad-frontend/src/components/Staff/Staff.js
import StaffCard from "../Cards/StaffCard";
import Columns from "../UI/Columns";

function Staff(props) {
  const staff = props.staff.map((e) => (
    <StaffCard
      key={e.name}
      picture={e.picture}
      name={e.name}
      degree={e.degree}
      description={e.description}
    />
  ));
  return <Columns columns="3">{staff}</Columns>;
}

export default Staff;

File: bitad-api/BitadAPI/Models/Staff.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BitadAPI.Models
{
    public class Staff
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Picture { get; set; }
        public string Description { get; set; }
        public string Degree { get; set; }
        public string Contact { get; set; }
        public bool IsPublic { get; set; }
        public string StaffRole { get; set; }

    }
}
